### PR TITLE
Switch to jbuilder

### PR DESCRIPTION
**Pull request by samoht**

### CODE REVIEW

**Comment by samoht** [comment]
See discussion on #3

File: lazy-trie.opam [hunk 1]
```diff
@@ -3,20 +3,15 @@ maintainer: "thomas@gazagnaire.org"
...
-  ["ocaml" "setup.ml" "-build"]
+  ["jbuilder" "subst"] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
 ]
-remove: [
-  ["ocamlfind" "remove" "lazy-trie"]
-]
-build-doc: [ "ocaml" "setup.ml" "-doc" ]
 depends: [
   "ocamlfind"
```
Comment (yomimono) [inline lazy-trie.opam]: Does it still need `ocamlfind`?

Comment (samoht) [reply lazy-trie.opam]: Good point, fixed

**Comment by samoht** [comment]
ping @mirage/core

**Comment by hannesm** [comment]
thanks for your PR, merged.  in case you've some spare time, it would be very nice to have travis setup for this repository as well.

**Pull request merged**